app/content/views/event.py

```diff
@@ -110,20 +113,35 @@ def update(self, request, pk):
             )
 
     def create(self, request, *args, **kwargs):
+        if (
+            "is_paid_event" in request.data
+            and request.data["is_paid_event"]
+            and "paid_information" not in request.data
+        ):
+            request.data["paid_information"] = {}
+            request.data["paid_information"]["price"] = 0.00
+            request.data["paid_information"]["paytime"] = time(second=0)
+
```

Comment:
Har vi lyst til å sette default verdier her om det ikke blir gitt? er det ikke litt dumt om det er 0 sekunder på time isåfall ?

Reply:
Jeg er for så vidt enig. Det var Martin som kom med forslaget om å ha en default hvis man ville opprette et betalt arrangement  uten å sette inn tid og pris enda. Men vi har ikke gjort det mulig for å gjøre dette i frontend, så enig i at den kan fjernes. Hvis man ikke vet hvilken pris og tid man vil bruke kan man bare opprette et vanlig først og så endre, siden jeg har tilrettelagt for denne muligheten.